test/smoke/packages/cert-manager/1.5.3/cert-manager-test.sh

```diff
@@ -73,17 +82,18 @@ spec:
     - www.example.com
 EOF
 
-sleep 45s
+timeout=300
+while [[ $(kubectl get certificate/example-com -n ${NAMESPACE} -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" && ${timeout} -ne 0 ]]; do echo "waiting for certificate" && sleep 1 && timeout=${timeout}-1 ; done
 
-status="$(kubectl get certificate/example-com | tail -n 1 | awk '{print $2}')"
-echo "${status}"
-tanzu package installed delete cert-manager -y
+status="$(kubectl get certificate/example-com -n ${NAMESPACE} | tail -n 1 | awk '{print $2}')"
```

Comment:
We could use `kubectl get certificate/example-com -n ${NAMESPACE} -o jsonpath={...}` here too, similar to https://github.com/vmware-tanzu/community-edition/pull/2517/files#diff-693b938505f694eb48282d5b45e5c3bd1e6e17363b0a7b9e7198b5fdf9115e18R86